assignment-1/time_pipe.c

```diff
@@ -0,0 +1,59 @@
+#include <sys/types.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <iostream>
+#include <sys/wait.h>
+#include <sys/time.h>
+
+
+struct timeval_t
+{
+
+  time_t tv_sec;
+  suseconds_t tv_usec;
+
+timeval(int *elapsed_time)
+{
+  tv_sec = elapsed_time;
+  tv_usec = elapsed_time;
+
+}

```

Comment:
I believe he made a mistake in the guidelines by suggesting `timeval_t` when it should be `timeval`. The [manpage](http://manpages.ubuntu.com/manpages/impish/man2/gettimeofday.2.html) on `gettimeofday()` says to use the `timeval` struct. So I don't think we should create a new struct for this.